libs/remix-ui/file-explorer/src/lib/file-explorer.tsx

```diff
@@ -557,7 +557,7 @@ export const FileExplorer = (props: FileExplorerProps) => {
   }
 
   const publishToGist = () => {
-    modal('Create a public gist', 'Are you sure you want to publish all your files in browser directory anonymously as a public gist on github.com? Note: this will not include directories.', {
+    modal('Create a public gist', `Are you sure you want to publish all your files in ${name} directory anonymously as a public gist on github.com? Note: this will not include directories.`, {
```

Comment:
I think the word **directory** is wrong and should be replaced with 'workspace'

so currently if my workspace is named Fred it reads: 
Are you sure you want to publish all your files in Fred directory anonymously as a public gist on github.com?

but as a user I don't see the workspace as a directory.  So maybe this is better - (also I moved the word anonymously)

Are you sure you want to anonymously publish all your files in the Fred workspace as a public gist on github.com?

So it would be more like this:
Are you sure you want to anonymously publish all your files in the ${name} workspace as a public gist on github.com? Note: this will not include directories.